src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -476,29 +476,34 @@ public VisualEditorEditTemplateDialog openEditTemplateDialog() {
   }
 
   public VisualEditorPageObject clickInsertToolButton() {
+    wait.forElementClickable(insertDropdownMenuButton);
     insertDropdownMenuButton.click();
     return this;
   }
 
   public VisualEditorPageObject clickInsertInfoboxFromInsertToolMenu() {
-    wait.forElementVisible(infoboxInDropdownMenu);
+    wait.forElementClickable(infoboxInDropdownMenu);
     infoboxInDropdownMenu.click();
     return this;
   }
 
   public VisualEditorPageObject selectInfoboxTemplate(int i) {
-    infoboxTemplatesList.get(i).click();
+    WebElement element = infoboxTemplatesList.get(i);
+    wait.forElementClickable(element);
+    element.click();
```

Comment:
can we add here more descriptive name instead of `element`? Maybe `templateName`, `templateElement` or `template`?